src/main/java/uk/gov/digital/ho/hocs/info/api/MemberService.java

```diff
@@ -49,7 +49,6 @@ void updateWebMemberLists() {
         updateMember(listConsumerService.createCommonsFromUKParliamentAPI());
         updateMember(listConsumerService.createLordsFromUKParliamentAPI());
         updateMember(listConsumerService.createFromIrishAssemblyAPI());
-        updateMember(listConsumerService.createFromEuropeanParliamentAPI());
```

Comment:
Might as well remove anything to do with the European Parliament API. We won't require this I shouldn't have thought, plus if we need to revert it will be in Git history.